orcestra/postprocess/level1.py

```diff
@@ -123,21 +123,26 @@ def _trim_dataset(ds, dim="time"):
     return trimmed_ds
 
 
-def coarsen_radiometer(ds):
-    """Coarsen radiometer data to 1 Hz.
+def _filter_spikes(ds):
+    """
+    Filters out spikes in a time series by comparing the difference between each point
+    and the minimum of the surrounding 5 minutes.
 
     Parameters
     ----------
-    ds : xr.Dataset
-        Level0 radiometer dataset.
+    ds : xr.DataArray
+        DataArray to filter.
 
     Returns
     -------
-    xr.Dataset
-        Radiometer data coarsened to 1 Hz.
+    xr.DataArray
+        Filtered DataArray.
     """
-
-    return ds.coarsen(time=4, boundary="pad").mean().drop_duplicates("time")
+    diff = ds - ds.rolling(time=4 * 60 * 5, center=True, min_periods=1).min()
+    threshold = diff.std() * 1
```

Comment:
Could you make both the interval and the threshold for outliers a user parameter? Defaults are fine, but I guess it would be nice to be able to adjust this

Reply:
Sure, is implemented now.